p2p/fechelper.go

```diff
@@ -0,0 +1,2133 @@
+package p2p
+
+import (
+	"fmt"
+
+	"github.com/aristanetworks/goarista/monotime"
+)
+
+//CFECHelper helper class for fec
+type CFECHelper struct {
+	canVec         [255]*VBitVec
+	bitLen, fecLen uint
+}
+
+//Init initialize helper.
+func (h *CFECHelper) Init(_bitLen uint, _fecLen uint) bool {
```

Comment:
This function is so long, and more duplicated logic, suggest to refine it.

Reply:
we will fix in future.